getting_started/introduction/reading_the_class_reference.rst

```diff
@@ -0,0 +1,111 @@
+.. _doc_reading_the_class_reference:
+
+Reading the Class Reference
+===========================
+
+Godot has over 200 unique Nodes at your disposal, and over 500 different types
+in total. As any tool of that size should, Godot offers a Class Reference to
+gather various important pieces of info about each class all in once place.
+
+Class Reference
+---------------
+
+The Godot Class Reference can be viewed online :ref:`here <doc_class_reference>`
+as well as directly within the editor. From the Script window, click "Search
+Help" (or press :kbd:`F1`) to open it.
+
+.. image:: img/editor-opening-class-reference.png
+
+From there, you can select any Node (or other type) in the engine to learn
+more about it.
+
+.. image:: img/class-reference-nodes-list.png
+
+Class Reference Page
+--------------------
+
+Each class's page includes the following information (a single given page
+may not have all of these):
+
+- Inherited class.
+- Inheriting class(es).
+- Description.
+- Tutorials.
+- Property List.
+- Method List.
+- Signals.
+- Enumerations.
+- Constants.
+- Property/Method Details.
+
+Inheritance
+-----------
+
+.. image:: img/class-ref-inheritance.png
+
+Godot make use of Object Oriented Programming (OOP), which allows one class to
+inherit the attributes of another. In this specific case, it can be seen that
+:ref:`CanvasItem <class_CanvasItem>` inherits :ref:`Node <class_Node>`, which in
+turn inherits :ref:`Object <class_Object>`, and also is inherited by :ref:`Node2D
+<class_Node2D>` and :ref:`Control <class_Control>`. In OOP terminology, a
+CanvasItem "is a" Node, and a Node2D "is a" CanvasItem.
+
+Description
+-----------
+
+This section describes what the class is, what it can be used for, as well as
+some general tips and recommendations.
+
+Tutorials
+---------
+
+Any specific tutorials that directly relate to this class may be linked here.
+
+.. note:: General Tutorials are not viewable from within the editor.
+
+Properties/Methods
+------------------
+
+.. image:: img/class-ref-property-method.png
+
+A quick listing of all of the properties and methods defined for **this** class
+comes next, with more detailed descriptions being lower down the page.
+
+.. note:: Only those defined specifically for **this** class are listed here
+   (this also applies to signals, enums, and constants). As Godot uses OOP,
+   other classes can inherit these fields, and this class can inherit from
+   others.
+
+   For example, if you look at the Node2D page to try to see how to make it
+   invisible, you wan't find that there, because the ``visible`` property is
```

Comment:
```suggestion
   invisible, you won't find that there, because the ``visible`` property is
```

I don't have time to read through everything today but this spelling error is why your PR is failing to build. I didn't see any others. In the future consider putting your changes into a program that has spellchecking like Word, Google Docs, or Libre Office Writer.